src/Form/tests/FormAutosuggest.test.jsx

```diff
@@ -21,26 +23,53 @@ function FormAutosuggestTestComponent(props) {
       name="FormAutosuggest"
       floatingLabel="floatingLabel text"
       helpMessage="Example help message"
-      errorMessageText="Example error message"
-      onSelected={props.onSelected}
+      valueRequiredErrorMessageText="Example value required error message"
+      selectionRequiredErrorMessageText="Example selection required error message"
+      customErrorMessageText="Example custom error message"
+      onChange={props.onChange}
+      valueRequired={props.valueRequired}
+      selectionRequired={props.selectionRequired}
+      customError={props.customError}
     >
-      <FormAutosuggestOption>Option 1</FormAutosuggestOption>
+      <FormAutosuggestOption id="option-1-id">Option 1</FormAutosuggestOption>
       <FormAutosuggestOption onClick={props.onClick}>Option 2</FormAutosuggestOption>
       <FormAutosuggestOption>Learn from more than 160 member universities</FormAutosuggestOption>
     </FormAutosuggestWrapper>
   );
 }
 
+function FormAutosuggestLabelTestComponent() {
+  return (
+    <FormGroup>
+      <FormLabel data-testid="autosuggest-label">
+        <h3>Label</h3>
+      </FormLabel>
+      <FormAutosuggestWrapper>
+        <FormAutosuggestOption>Option</FormAutosuggestOption>
+      </FormAutosuggestWrapper>
+    </FormGroup>
+  );
+}
+
 FormAutosuggestTestComponent.defaultProps = {
-  onSelected: jest.fn(),
+  onChange: jest.fn(),
   onClick: jest.fn(),
+  valueRequired: false,
+  selectionRequired: false,
+  customError: false,
 };
 
 FormAutosuggestTestComponent.propTypes = {
-  /** Specifies onSelected event handler. */
-  onSelected: PropTypes.func,
+  /** Specifies onChange event handler. */
+  onChange: PropTypes.func,
   /** Specifies onClick event handler. */
   onClick: PropTypes.func,
+  /** Specifies if empty values trigger an error state */
+  valueRequired: PropTypes.bool,
+  /** Specifies if freeform values trigger an error state */
+  selectionRequired: PropTypes.bool,
+  /** Specifies the control is in a consumer provided error state */
```

Comment:
[question]: Does it make sense to add additional comments to the details in the test file? It seems to me that we already have these descriptions on the page of the component itself.

Reply:
I was following the pattern of the existing ones, but I agree they felt redundant. I've removed them.